config/ci.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Does this schema work with the `07-draft` meta-schema, or only `2020-12`?

Reply:
From having a quick look, I think `definitions` in `07-draft` was changed to `$defs` in 2019-09, and `definitions` is now deprecated.  From what I can tell, that's the only semi-incompatible feature used in this schema.

Reply:
Coolios :)